### ISSUE TITLE
Remove titlebar and give all its functionality to the actionbar

### ISSUE THREAD

**Issue by josem11200** [papyros/papyros-shell#224]
The titlebar doesn't fit well in my opinion with a material desktop, and the action bar can do the job well.
##

**Comment by ctnlaring** [comment]
@josem11200 This causes problems with apps that aren't made with qml material. I think this would be cool but should be saved for a much later release because the vast majority of apps will use the normal titlebar so it makes sense to focus on that for now.

**Comment by ctnlaring** [comment]
Apps made in qml material can do this right now. The Liri apps all do this and it could be done for the core Papyros apps as well if somebody wants to do it.

**Comment by josem11200** [comment]
Yes, I know that it would cause inconsistencies between Papyros-dedicated apps and the rest. On the other hand, I didn't know that some apps do it already.

**Comment by josem11200** [comment]
(incorrect button :P )

**Comment by iBelieve** [comment]
This is definitely something we're going to implement fully. Apps not using our framework will use regular decorations, and apps built with our framework will be able to use it if they want. Right now we have initial non-complete support, as you can see in the Liri apps, but it needs to get polished and improved.

**Comment by ctnlaring** [comment]
@iBeliever And have them use the normal titlebar buttons. The ones the Liri apps use are pretty terrible IMHO

**Comment by josem11200** [comment]
Nice!
